helm/docs/common-configuration.md

```diff
@@ -0,0 +1,450 @@
+  # See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+  # ingressClassName: nginx
+```
+
+### Istio Ingress Gateway
+
+If you need to configure Bold BI with Istio then you can change the value as `istio` in your configuration like below.
+
+```console
+loadBalancer:
+  type: istio
+```
+
+### sticky session
+
+You can change the affinityCookieExpiration time. The default value is 600s.
+
+```console
+loadBalancer:
+  affinityCookie:
+    enable: true
+    affinityCookieExpiration: 600
+```
+
+### SSL Configuration
+
+If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, just pass your DNS with `https` protocol to `appBaseUrl`, by doing this it will automatically enable SSL in both Ingress and Istio.
+
+> **NOTE:**  You have to create the TLS Secret with name `bold-tls` or else change the secret name in your values.yaml
+
+Run the following command to create a TLS secret with your SSL certificate.
+
+```console
+# Ingress
+kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+
+# Istio
+kubectl create secret tls bold-tls -n istio-system --key <key-path> --cert <certificate-path>
+```
+
+### Map multiple domains
+
+You can map mutiple domains in both Ingress Nginx and Istio like below. While mapping multiple domains you have to include the `appBaseUrl` in any of the matching host array.
+
+FOr multiple domain scenerio the `singleHost` secret will not be considered, you have to mention your secret inside `multipleHost` section.
+
+`Ingress Nginx`
+
+```console
+loadBalancer:
+  singleHost:
+    secretName: bold-tls
+
+  multipleHost:
+    hostArray:
+      - hosts: 
+          - cd1.abc.com
+          - cd2.abc.com
+        secretName: tls-secret1
+      - hosts: 
+          - cd1.xyz.com
+          - cd2.xyz.com
+        secretName: tls-secret2
+```
+
+`Istio Ingress Gateway`
+
+```console
+loadBalancer:
+  singleHost:
+    secretName: bold-tls
+
+  multipleHost:
+    hostArray:
+      - hosts: 
+          - cd1.abc.com
+          - cd2.abc.com
+        secretName: tls-secret
+```
+
+## Auto Scaling
+
+By default autoscaling is enabled in Bold BI, to disable autoscaling please set `autoscaling.enabled=false`.
+
+```console
+autoscaling:
+  enabled: true
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+  behavior:
+    stabilizationWindowSeconds: 60
+    podsValue: 1
+    podsPeriodSeconds: 60
+    percentValue: 10
+    percentPeriodSeconds: 60
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.enabled
+      </td>
+      <td>
+       By default the autoscaling will be enabled. turn this to <i>false</i> to disable the autoscaling functionality.
```

Comment:
By default, autoscaling will be enabled. turn this to <i>false</i> to disable the autoscaling functionality.

Reply:
concern addressed